gtfu/src/main/java/gtfu/tools/BlockDataGenerator.java

```diff
@@ -188,11 +198,12 @@ public static void main(String[] arg) throws Exception {
 
         if (agencyID == null) usage();
 
-        new BlockDataGenerator(cacheFolder, outputFolder, agencyID, date, uploadToGcloud);
+        new BlockDataGenerator(cacheFolder, outputFolder, agencyID, gtfsURL, date, uploadToGcloud);
     }
 
     class TripRecord {
         public String id;
+        public String headSign;
```

Comment:
very minor but it's just one one word, so camelcase is unnecessary. Just "headsign"